daemon/main.c

```diff
@@ -541,7 +541,7 @@ static void get_netdata_configured_variables() {
     netdata_configured_web_dir          = config_get(CONFIG_SECTION_GLOBAL, "web files directory",    netdata_configured_web_dir);
     netdata_configured_cache_dir        = config_get(CONFIG_SECTION_GLOBAL, "cache directory",        netdata_configured_cache_dir);
     netdata_configured_varlib_dir       = config_get(CONFIG_SECTION_GLOBAL, "lib directory",          netdata_configured_varlib_dir);
-    netdata_configured_home_dir         = config_get(CONFIG_SECTION_GLOBAL, "home directory",         netdata_configured_home_dir);
+    netdata_configured_home_dir         = config_get(CONFIG_SECTION_GLOBAL, "home directory",         getenv("HOME"));
```

Comment:
If `getenv("HOME")` is empty it should default to `netdata_configured_home_dir` which is `CACHE_DIR`.

Reply:
Sure, lemme do that.

Reply:
Is there a particular reason we're using the cache directory here instead of `/var/lib/netdata` like we configure as the actual home directory for the Netdata user?